dags/inference/maxtext_inference.py

```diff
@@ -37,84 +37,126 @@
   test_name_prefix = "maxtext-inference"
   test_models = {
       "llama2-7b": {
-          "per_device_batch_sizes": [1, 2, 4],
+          "sleep_time": 120,
+          "tpu_version_cores": [(TpuVersion.V5E, 8), (TpuVersion.V5P, 8)],
           "checkpoint": "gs://inference-benchmarks/models/llama2-7b/2024-04-25-14-01/param-only-decode-ckpt-maxtext/checkpoints/0/items",
           "maxtext_logs": "gs://inference-benchmarks/models/llama2-7b/2024-04-25-14-01/",
           "tokenizer": "tokenizer.llama2",
+          "per_device_batch_sizes": [1, 2, 4],
           # (ici_fsdp_parallelism, ici_autoregressive_parallelism, ici_tensor_parallelism)
           "ici_parallelisms": [(1, -1, 1), (1, 1, -1)],
+          "request_rate": 5,
+          "max_prefill_predict_length": 1024,
+          "max_target_length": 2048,
       },
       "llama2-13b": {
-          "per_device_batch_sizes": [1, 2],
+          "sleep_time": 120,
+          "tpu_version_cores": [(TpuVersion.V5E, 8), (TpuVersion.V5P, 8)],
           "checkpoint": "gs://inference-benchmarks/models/llama2-13b/2024-04-25-14-01/param-only-decode-ckpt-maxtext/checkpoints/0/items",
           "maxtext_logs": "gs://inference-benchmarks/models/llama2-13b/2024-04-25-14-01/",
           "tokenizer": "tokenizer.llama2",
+          "per_device_batch_sizes": [1, 2],
+          # (ici_fsdp_parallelism, ici_autoregressive_parallelism, ici_tensor_parallelism)
+          "ici_parallelisms": [(1, -1, 1), (1, 1, -1)],
+          "request_rate": 5,
+          "max_prefill_predict_length": 1024,
+          "max_target_length": 2048,
+      },
+      "llama2-70b": {
+          "sleep_time": 240,
+          "tpu_version_cores": [(TpuVersion.V5P, 8)],
+          "per_device_batch_sizes": [2, 4],
+          "checkpoint": "gs://inference-benchmarks/models/llama2-70b-chat/2024-05-08-23-16/param-only-decode-ckpt-maxtext/checkpoints/0/items",
+          "maxtext_logs": "gs://inference-benchmarks/models/llama2-70b-chat/2024-05-08-23-16/",
+          "tokenizer": "tokenizer.llama2",
           # (ici_fsdp_parallelism, ici_autoregressive_parallelism, ici_tensor_parallelism)
           "ici_parallelisms": [(1, -1, 1), (1, 1, -1)],
```

Comment:
Is this going to make two runs -- one with (1, -1, 1) and another with (1, 1, -1)?

Reply:
Yes.